lib/components/estate_card.dart

```diff
@@ -1,18 +1,24 @@
+import 'package:caderneta_campo_digital/models/propriedade.dart';
+import 'package:caderneta_campo_digital/pages/estate/estate_page.dart';
 import 'package:caderneta_campo_digital/utils/utils.dart';
 import 'package:flutter/material.dart';
 
 class EstateCard extends StatelessWidget {
-  final String estateName;
-  final String estateAddress;
-  final int qtdPlot;
+  final Propriedade estate;
 
   const EstateCard({
     Key? key,
-    required this.estateName,
-    required this.estateAddress,
-    required this.qtdPlot,
+    required this.estate,
   }) : super(key: key);
 
+  String getAddress() {
+    return estate.cidade + ' ' + estate.estado + ' cep: ' + estate.cep;
```

Comment:
Acredito que a formatação do card fique melhor assim:

```suggestion
    return estate.cidade + ' - ' + estate.estado + '\nCEP: ' + estate.cep;
```